TeamsSampleHub/README.md

```diff
@@ -1,35 +1,35 @@
-Microsoft Teams HOME app overview
+Microsoft Teams Sample hub app overview
```

Comment:
Remove "app" across the doc. it should be "Teams Sample Hub", in this case Teams Sample Hub overview